pipeline-runner/R/gem2s-1-download_cellranger.R

```diff
@@ -1,29 +1,36 @@
-download_cellranger <- function(input, pipeline_config) {
+#' Download cellranger files from S3
+#'
+#' @param input The input object from the request
+#' @param pipeline_config result of \code{load_config}
+#' @param prev_out \code{list()} that is added to in each with gem2s task.
+#'
+#' @return list where 'output' slot has config used by \code{load_cellranger}
+#' @export
+#'
+download_cellranger <- function(input, pipeline_config, prev_out = list()) {
   project_id <- input$projectId
   sample_names <- input$sampleNames
   sample_uuids <- input$sampleIds
 
-  print_config(1, "Cellranger", input, pipeline_config, list())
-
   s3 <- paws::s3(config = pipeline_config$aws_config)
 
-  fnames <- c("features.tsv.gz", "barcodes.tsv.gz", "matrix.mtx.gz")
+  cr_fnames <- c("features.tsv.gz", "barcodes.tsv.gz", "matrix.mtx.gz")
```

Comment:
what does `cr_fnames` stand for?

Reply:
cellranger filenames

Reply:
I've gone ahead and changes to `cellranger_fnames`